doc/source/tutorial/stats/continuous_sine.rst

```diff
@@ -0,0 +1,19 @@
+
+.. _continuous-sine:
+
+Sine Distribution
+===================
+
+Polar-angle distribution required for ensuring uniform distribution over a sphere.
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*} f\left(x\right) & = & \frac{1}{2}\sin \left(x\right) I_{\left[0,\pi\right]}\left(x\right)\\ F\left(x\right) & = & \frac{1}{2}\left[1-\cos(x)\right]I_{\left[0,\pi\right]}\left(x\right)+I_{\left(\pi,\infty\right)}\left(x\right)\\ G\left(\alpha\right) & = & 2\sin^{-1}\left(\sqrt{x}\right)\\ M\left(t\right) & = & \frac{1+e^{\pi t}}{2\left(1+t^2\right)}\\ \mu=m_{d}=m_{n} & = & \frac{\pi}{2}\\ \mu_{2} & = & \frac{\pi^2}{4}-2\\ \gamma_{1} & = & 0\\ \gamma_{2} & = & -2\frac{\pi^4-96}{\left(\pi^2-8\right)^2}\end{eqnarray*}
```

Comment:
Can this be written in a more readable way :)

Reply:
Indeed 😃 

```latex

\begin{align*}
f(x) & = \frac{1}{2}\sin{(x)} I_{[0,\pi]}(x)\\
F(x) & = \frac{1}{2}[1-\cos(x)]I_{[0,\pi]}(x)+I_{(\pi,\infty)}(x)\\
G(\alpha) & = 2\sin^{-1}(\sqrt{x})\\
M(t) & = \frac{1+e^{\pi t}}{2(1+t^2)}\\
\mu=m_{d}=m_{n} & = \frac{\pi}{2}\\
\mu_{2} & = \frac{\pi^2}{4}-2\\
\gamma_{1} & = 0\\
\gamma_{2} & = -2\frac{\pi^4-96}{(\pi^2-8)^2}
\end{align*}
```
If possible please avoid using `eqnarray` completely.

Reply:
Done. I was merely following the format in cosine. I thought it seemed a little odd to have that as the convention.